lib/commands/build.js

```diff
@@ -10,9 +9,9 @@ module.exports = Command.extend({
   aliases: ['b'],
 
   availableOptions: [
-    { name: 'environment', type: String,  default: 'development', aliases: ['e', { 'dev': 'development' }, { 'prod': 'production' }] },
-    { name: 'output-path', type: path,    default: 'dist/',       aliases: ['o'] },
-    { name: 'watch',       type: Boolean, default: false,         aliases: ['w'] },
+    { name: 'environment', type: String,             default: 'development', aliases: ['e', { 'dev': 'development' }, { 'prod': 'production' }] },
+    { name: 'output-path', type: function Path() {}, default: 'dist/',       aliases: ['o'] },
```

Comment:
while fixing tests it turned out that quite a few places expect the type to be a constructor and rely on `type.name` to exist or check `typeof type === 'function'` (not only the output formatter but also validations and the logic that decides to use the default value or not and so on).

So the easiest way is to let it be a function with a proper name and be consistent with all the other types available.